go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -210,14 +210,19 @@ func runOvnKube(ctx *cli.Context) error {
 			if runtime.GOOS == "windows" {
 				panic("Windows is not supported as master node")
 			}
-			// run the cluster controller to init the master
-			err := clusterController.StartClusterMaster(master)
-			if err != nil {
-				logrus.Errorf(err.Error())
-				panic(err.Error())
+
+			manageDBServers := ctx.Bool("manage-db-servers")
+			// Check if the pod ip is set or not if manageDBServers is set
+			if manageDBServers {
+				if ip := net.ParseIP(config.Kubernetes.OVNKubePodIP); ip == nil {
```

Comment:
@numansiddique we should parse and fail invalid OVNKubePodIP in config.go

Reply:
Done